catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/security/WorkspacePreIngestPlugin.java

```diff
@@ -71,38 +52,6 @@ public WorkspacePreIngestPlugin(WorkspaceSecurityConfiguration config, EventAdmi
         .collect(Collectors.toMap(Metacard::getId, m -> m));
   }
 
-  protected Map<String, SortedSet<String>> getSubjectAttributes() {
-    return SubjectUtils.getSubjectAttributes(SecurityUtils.getSubject());
-  }
-
-  private SortedSet<String> getSubjectAttribute(String key) {
-    Map<String, SortedSet<String>> attrs = getSubjectAttributes();
-
-    if (attrs.containsKey(key)) {
-      return attrs.get(key);
-    }
-
-    return Collections.emptySortedSet();
-  }
-
-  protected String getSubjectName() {
-    return SubjectUtils.getName(SecurityUtils.getSubject());
-  }
-
-  protected void alertMissingClaim(String ownerAttribute) {
```

Comment:
Do we no longer want to alert the admin when a subject is missing the configured identity claim?

If so, we should remove this [dependency](https://github.com/codice/ddf/pull/2309/files#diff-c43f02696e0d61da72469f60e6832731R255) which was added for alerting?

Reply:
Unless others prefer different behavior, since there is fall back logic for email and then finally username we should always have an id that we are returning.